src/lib.rs

```diff
@@ -59,7 +60,7 @@ extern crate flate2;
 extern crate num_traits;
 extern crate safe_transmute;
 
-#[cfg(feature = "nalgebra_affine")] mod affine;
+#[cfg(feature = "nalgebra_affine")] pub mod affine;
```

Comment:
Just raising the question: does it make sense for this module to become part of the crate's public API?